test/deploy-eks-cluster.sh

```diff
@@ -21,6 +21,22 @@ if [[ -z "${EKS_CLUSTER_K8_VERSION}" ]]; then
 fi
 
 function deleteCluster() {
+  echo "Cleanup role, security-group, open-id ${TEST_CLUSTER_NAME}"
+  account_id=$(aws sts get-caller-identity --query "Account" --output text)
+  rolename=$(echo ${TEST_CLUSTER_NAME} | awk -F- '{print "EBS_" $(NF-1) "_" $(NF)}')
+  role_attached_policies=$(aws iam list-attached-role-policies --role-name $rolename --query 'AttachedPolicies[*].PolicyArn' --output text)
+  for policy_arn in ${role_attached_policies};
+  do
+    aws iam detach-role-policy --role-name ${rolename} --policy-arn ${policy_arn}
+  done
+
+  aws iam delete-role --role-name ${rolename}
+  oidc_id=$(aws eks describe-cluster --name ${TEST_CLUSTER_NAME} --query "cluster.identity.oidc.issuer" --output text | cut -d '/' -f 5)
+  aws iam delete-open-id-connect-provider --open-id-connect-provider-arn arn:aws:iam::${account_id}:oidc-provider/${oidc_id}
```

Comment:
I believe this has to be:
`aws iam delete-open-id-connect-provider --open-id-connect-provider-arn arn:aws:iam::${account_id}:oidc-provider/oidc.eks.us-west-2.amazonaws.com/id/${oidc_id}`

I tested this out manually